agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -964,3 +964,17 @@ func TestTwoTasksSharedLocalVolume(t *testing.T) {
 	rExitCode, _ := rTask.ContainerExitcode("task-shared-vol-read")
 	assert.Equal(t, 42, rExitCode, fmt.Sprintf("Expected exit code of 42; got %d", rExitCode))
 }
+
+// TestHostnameInAWSVPCMode tests the hostname was correctly set to the private
```

Comment:
Instead of writing this in code, can you use the simpletests generator and a test definition in the simpletests_unix folder?